tests/service/commandHandler.spec.ts

```diff
@@ -7,6 +7,7 @@ describe('Command Handler', () => {
         const cmdInst = new CommandHandler();
         expect(cmdInst.commandLoader()).to.be.an('array');
     });
+
```

Comment:
change the test to:
```suggestion
it('command object check.', () => {
        const cmdInst = new CommandHandler();
        const cmdList = cmdInst.commandLoader();
        for (let cmd of cmdList) {
            for (let key of ['execute', 'name', 'alias', 'description', 'man']) {
                expect(cmd).to.have.property(key);
            }
        }
    });
```

and add a man variable on help which has same value as description.

Reply:
changed the test as suggested